packages/config/config/devices/0x027a/zen71.json

```diff
@@ -76,21 +76,36 @@
 			]
 		},
 		"3": {
-			"label": "Auto Turn-Off Timer (Minutes)",
-			"description": "How long the switch remains on",
+			"label": "Auto Turn-Off Timer",
+			"description": "Set the number of minutes to automatically turn off switch once it has been turned on.",
```

Comment:
This description is unnecessary. Please remove them throughout the files if the label makes clear what it does. We already define it is in minutes via the displayed unit. This is a broader problem in the original files, so if you can help us clean that up in these we appreciate it.

Reply:
Sure. I'll just go all out and templatize all these, at least the Zen line.